scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -162,6 +162,11 @@ private fun generateScannerOptions(options: JsonNode?): String {
     }.orEmpty()
 }
 
+private fun getInputPath(result: JsonNode): String {
```

Comment:
commit-msg: Shoud "being" be dropped?

Reply:
The sentence sounds less proper to me when omitted it.